testassets/HelloWorldApp/wwwroot/index.html

```diff
@@ -7,6 +7,10 @@ <h1>Hello</h1>
     <p>
         <button onclick="callDotNet()">Call .NET</button>
     </p>
+    
+    <p>
+        <a href="http://www.microsoft.com">Link to Microsoft</a>
```

Comment:
Not a big deal but I'd personally prefer:

```suggestion
        <a href="http://www.example.com">Link to example.com</a>
```